src/wxWidgets.cpp

```diff
@@ -242,7 +246,7 @@ void dxfv_wxWidgetsFrame::OnSelectSolidParser(wxCommandEvent& event)
 void dxfv_wxWidgetsFrame::OnPaint(wxPaintEvent& event)
 {
     wxPaintDC dc(this);
-    dxf->DC( &dc );
+    theDC = &dc; // set the global wxDC pointer
```

Comment:
Is this a memory leak?

The pointer to the old DC is being overwritten by the pointer to the new DC.  So this will work until memory is exhausted.  This could be quite quickly if the pan and zoom features are used frequently.  AFAIK the old DC should be deleted before the pointer is overwritten.

Reply:
Oops, my mistake.  dc is an automatic variable, so it is deleted when OnPaint() exits.  Everything is fine.

If you create a pull request with just this commit, I will accept it.